packages/ketchup/src/components/kup-activity-timeline/kup-activity-timeline-declarations.ts

```diff
@@ -0,0 +1,42 @@
+import { KupDataColumn, KupEventPayload } from '../../components';
+import { KupDataRow } from '../../managers/kup-data/kup-data-declarations';
+
+export enum KupActivityTimelineProps {
+    data = 'Dataset containing the activities list',
+    dateColumn = 'Column containing dates',
+    timeColumn = 'Column containing times',
+}
+
+export interface KupActivityTimelineData {
+    title: string;
+    value: string;
+    columnName: string;
+    cellId: string;
+}
+
+export interface KupActivityTimelineActivity {
+    time: string;
+    columns: KupActivityTimelineData[];
+}
+
+export interface ActivityTimeline {
```

Comment:
Wrong naming convention.

Reply:
if we change this interface name to KupActivityTimeline, it will conflict with the the component class name which is also KupActivityTimeline, could you suggest a suggestion for naming convention in this case?

Reply:
Fxied